dev-libs/hyperscan/hyperscan-5.1.1.ebuild

```diff
@@ -24,16 +24,13 @@ BDEPEND="${RDEPEND}"
 
 REQUIRED_USE="cpu_flags_x86_ssse3 ${PYTHON_REQUIRED_USE}"
 
-src_prepare() {
-	# upstream workaround
-	append-cxxflags -Wno-redundant-move
-	cmake-utils_src_prepare
-}
+PATCHES=( "${FILESDIR}/gentoo_build_is_release.patch" )
```

Comment:
Wouldn't it actually be better to use a sane build type than patch upstream build system to accept historical Gentoo insanity?

Reply:
And override CFLAGS with "sane defaults"?

Reply:
We override the CFLAGS override.

Reply:
with custom build type?

Reply:
With CMake override for all standard build types.

Reply:
Docs on `cmake-utils` eclass don't contain anything about it